backend/store/etcd/event_store_test.go

```diff
@@ -112,3 +112,24 @@ func TestEventStorage(t *testing.T) {
 		assert.Error(t, err)
 	})
 }
+
+func TestDinnaeStoreMetrics(t *testing.T) {
```

Comment:
I don't think I'm smart enough to know what `Dinnae` means..

Reply:
https://en.wikipedia.org/wiki/Scotticism

Reply:
I had to consult that wiki page too :(
Should it just read `TestDoNotStoreMetrics()`?